bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -84,13 +79,15 @@ else if (!isRegistered(userService.getCurrentUser().getUserId()) && !request.get
 }
 
   public boolean isRegistered(String id) {
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    Query query = new Query("User")
-      .setFilter(new Query.FilterPredicate(
-        "id", Query.FilterOperator.EQUAL, id));
-    PreparedQuery results = datastore.prepare(query);
-    Entity entity = results.asSingleEntity();
-    return entity != null;
+    // DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Query<User> userQuery = ObjectifyService.ofy().load().type(User.class);
+    List<User> allUsers = userQuery.list();
+
+    List<User> isRegistered =
```

Comment:
Name this something else.. right now it sounds like a boolean. "registeredUsers" maybe? Why would there be multiple?

Reply:
renamed